bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/package.scala

```diff
@@ -50,7 +50,7 @@ package object core {
     /** Construct an exact or interpolated slice. */
     def iSlice(lo: K, hi: K): Option[K ==>> V] =
       ^(iLookup(lo), iLookup(hi)) { (lov, hiv) =>
```

Comment:
`^(a, b)(f)` is another way to write `(a |@| b)(f)` ... not sure why I did it that way.